src/handlers/sqs-indexer/handle-bridge-out-event.ts

```diff
@@ -0,0 +1,223 @@
+import {TransactionReceipt, TransactionResponse} from '@ethersproject/abstract-provider'
+import {getNetworkByHolographId} from '@holographxyz/networks'
+import {TransactionDescription} from '@ethersproject/abi'
+import {Environment} from '@holographxyz/environment'
+
+import {create2address, decodeDeploymentConfig, DeploymentConfig} from '../../utils/contract-deployment'
+import {BridgeOutArgs, BridgeOutErc20Args, decodeBridgeOutErc20Args} from '../../utils/bridge'
+import {networkToChainId, sha3, storageSlot, toAscii} from '../../utils/utils'
+import {NetworkMonitor} from '../../utils/network-monitor'
+import {BridgeDirection, ContractType, PayloadType, SqsMessageBody} from '../../types/sqs'
+import SqsService from '../../services/sqs-service'
+import {hexZeroPad} from '@ethersproject/bytes'
+
+async function handleBridgeOutEvent(
+  networkMonitor: NetworkMonitor,
+  environment: Environment,
+  transaction: TransactionResponse,
+  network: string,
+  tags: (string | number)[],
+): Promise<void> {
+  const receipt: TransactionReceipt | null = await networkMonitor.getTransactionReceipt({
```

Comment:
So I would like a lot of the logic in this function to be in the processor. This function should be super small and basically send a job to the queue.